polaris-core/src/main/resources/schemas/policies/system/orphan-file-removal/2025-02-03.json

```diff
@@ -0,0 +1,47 @@
+{
+  "license": "Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)",
+  "$id": "https://polaris.apache.org/schemas/policies/system/orphan-file-removal/2025-02-03.json",
+  "title": "Orphan File Removal Policy",
+  "description": "Inheritable Polaris policy schema for Iceberg table orphan file removal.",
+  "type": "object",
+  "properties": {
+    "version": {
+      "type": "string",
+      "const": "2025-02-03",
+      "description": "Schema version."
+    },
+    "enable": {
+      "type": "boolean",
+      "description": "Enable or disable orphan file removal."
+    },
+    "max_orphan_file_age_in_days": {
+      "type": "number",
+      "description": "Specifies the maximum age (in days) for orphaned files before they are eligible for removal."
+    },
+    "location": {
+      "type": "string",
+      "description": "Specifies a custom directory to search for files instead of the default table location. Use with caution—if set to a broad location (e.g., s3://my-bucket instead of s3://my-bucket/my-table-location), all unreferenced files in that path may be permanently deleted, including files from other tables. Following best practices, tables should be stored in separate locations to avoid accidental data loss."
```

Comment:
This hard to read in an IDE like IntelliJ as it is a single long line. Json doesn't support a way to break one line to multiple lines. This makes me think we may use the format `yaml` instead of `json`.

Reply:
The alternative is just to put line breaks in the string, and then preprocess it anywhere we want to strip out the whitespace

Reply:
> put line breaks in the string
One of my versions was that :). It didn't help much, esp. in the IDE.